WalletWasabi/CoinJoin/Client/Clients/CoinJoinClient.cs

```diff
@@ -218,9 +218,12 @@ private async Task TryProcessStatusAsync(IEnumerable<RoundStateResponseBase> sta
 					// First, if there's delayed round registration update based on the state.
 					if (DelayedRoundRegistration is { })
 					{
-						ClientRound roundRegistered = State.GetSingleOrDefaultRound(DelayedRoundRegistration.AliceClient.RoundId);
-						roundRegistered.Registration = DelayedRoundRegistration;
-						DelayedRoundRegistration = null; // Do not dispose.
+						ClientRound? roundRegistered = State.GetSingleOrDefaultRound(DelayedRoundRegistration.AliceClient.RoundId);
+						if (roundRegistered is { })
```

Comment:
What should be done here if `roundRegistered` is `null`?

Reply:
Nothing.